akka-stream/src/main/scala/akka/stream/scaladsl/Hub.scala

```diff
@@ -443,6 +481,11 @@ private[akka] class BroadcastHub[T](bufferSize: Int)
                   callback.invoke(UnRegister(consumer.id, startFrom, startFrom)))
               case _ => ()
             }
+
+            if (initialized && pending.nonEmpty) {
+              pending.foreach(publish)
```

Comment:
They were placed in `pending` from `publish`, so it's OK to publish again now that we're initialized.

Might be slightly simpler if we put this into the `if (activeConsumers >= startAfterNrOfConsumers)` block above?